bin/impl/setup-fluo.sh

```diff
@@ -47,55 +27,8 @@ echo "Setting up Accumulo"
 echo "Setting up Fluo"
 "$FLUO_DEV"/bin/impl/setup-fluo-only.sh
 
-if [[ "$SETUP_METRICS" == "true" ]]; then
-  echo "Removing previous versions of InfluxDB & Grafana"
-  rm -rf "$INSTALL"/influxdb-*
-  rm -rf "$INSTALL"/grafana-*
-
-  echo "Remove previous log dirs and recreate"
-  rm -f "$LOGS_DIR"/metrics/*
-  mkdir -p "$LOGS_DIR"/metrics
-
-  echo "Setting up metrics (influxdb + grafana)..."
-  tar xzf "$DOWNLOADS"/build/"$INFLUXDB_TARBALL" -C "$INSTALL"
-  "$INFLUXDB_HOME"/bin/influxd config -config "$FLUO_DEV"/conf/influxdb/influxdb.conf > "$INFLUXDB_HOME"/influxdb.conf
-  if [[ ! -f "$INFLUXDB_HOME"/influxdb.conf ]]; then
-    echo "Failed to create $INFLUXDB_HOME/influxdb.conf"
-    exit 1
-  fi
-  $SED "s#DATA_DIR#$DATA_DIR#g" "$INFLUXDB_HOME"/influxdb.conf
-  rm -rf "$DATA_DIR"/influxdb
-  "$INFLUXDB_HOME"/bin/influxd -config "$INFLUXDB_HOME"/influxdb.conf &> "$LOGS_DIR"/metrics/influxdb.log &
-
-  tar xzf "$DOWNLOADS"/build/"$GRAFANA_TARBALL" -C "$INSTALL"
-  cp "$FLUO_DEV"/conf/grafana/custom.ini "$GRAFANA_HOME"/conf/
-  $SED "s#GRAFANA_HOME#$GRAFANA_HOME#g" "$GRAFANA_HOME"/conf/custom.ini
-  $SED "s#LOGS_DIR#$LOGS_DIR#g" "$GRAFANA_HOME"/conf/custom.ini
-  mkdir "$GRAFANA_HOME"/dashboards
-  cp "$FLUO_HOME"/contrib/grafana/* "$GRAFANA_HOME"/dashboards/
-  "$GRAFANA_HOME"/bin/grafana-server -homepath="$GRAFANA_HOME" 2> /dev/null &
-
-  echo "Configuring InfluxDB..."
-  sleep 10
-  "$INFLUXDB_HOME"/bin/influx -import -path "$FLUO_HOME"/contrib/influxdb/fluo_metrics_setup.txt
-
-  # allow commands to fail
-  set +e
-
-  echo "Configuring Grafana..."
-  echo "Adding InfluxDB as datasource"
-  sleep 10
-  retcode=1
-  while [[ $retcode != 0 ]];  do
-    curl 'http://admin:admin@localhost:3000/api/datasources' -X POST -H 'Content-Type: application/json;charset=UTF-8' \
-      --data-binary '{"name":"fluo_metrics","type":"influxdb","url":"http://localhost:8086","access":"direct","isDefault":true,"database":"fluo_metrics","user":"fluo","password":"secret"}'
-    retcode=$?
-    if [[ $retcode != 0 ]]; then
-      echo "Failed to add Grafana data source.  Retrying in 5 sec.."
-      sleep 5
-    fi
-  done
-fi
+echo "Setting up Metrics"
+"$FLUO_DEV"/bin/impl/setup-metrics.sh
```

Comment:
Seems like it would be a little cleaner to move the check for SETUP_METRICS being true to here.